### PR TITLE
feat: concurrent userops & example

### PR DESCRIPTION
**Pull request by 0xSulpiride**
## Description

- feat: `key` param in `sdk.estimate` function to make use all values of semi-abstraced nonce that allows to send many userops from one wallet at once (https://eips.ethereum.org/EIPS/eip-4337#semi-abstracted-nonce-support)

## Types of changes
What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Further comments (optional)

-

### CODE REVIEW

**Review by ch4r10t33r — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@etherspot/prime-sdk",
-  "version": "1.4.2",
+  "version": "1.5.2",
```
Comment (vignesha22) [inline package.json:3]: Can you change this to 1.5.0 please

**Review by vignesha22 — CHANGES_REQUESTED** [review]
Just a minor change on versioning

**Review by arddluma — APPROVED** [review]

**Review by vignesha22 — APPROVED** [review]
LGTM

**Pull request merged**